forge-gui/res/cardsfolder/upcoming/jhoira_ageless_innovator.txt

```diff
@@ -0,0 +1,9 @@
+Name:Jhoira, Ageless Innovator
+ManaCost:U R
+Types:Legendary Creature Human Artificer
+PT:2/3
+A:AB$ PutCounter | Cost$ T | CounterType$ INGENUITY | CounterNum$ 2 | SubAbility$ DBChangeZone | StackDescription$ {p:You} puts two ingenuity counters on CARDNAME, then {p:You} may put an artifact card with mana value X or less from hand onto the battlefield, where X is the number of ingenuity counters on NICKNAME. | SpellDescription$ Put two ingenuity counters on CARDNAME, then you may put an artifact card with mana value X or less from your hand onto the battlefield, where X is the number of ingenuity counters on NICKNAME.
```

Comment:
you should add INGENUITY counters to CounterEnumType.java
you get to pick the color :)

Reply:
`INGENUITY("INGTY", 99, 67, 205)`

My contribution to color theory! I can add the addition to the next PR :)

Reply:
it would be fine in this one, too... but if it's already on another branch no need to go through the hassle 😁